src/main/cpp/subsys/SwerveChassis.cpp

```diff
@@ -467,12 +477,20 @@ void SwerveChassis::AdjustRotToMaintainHeading
 }
 
 void SwerveChassis::AdjustRotToPointTowardGoal
-(
-    units::radians_per_second_t& rot 
+(   
+    Pose2d                     robotPose,
+    Pose2d                     goalPose, 
+    units::meters_per_second_t &xSpeed,
+    units::meters_per_second_t &ySpeed,
+    units::radians_per_second_t &rot     
 )
 {
-    auto myPose = GetPose();
-
+    Pose2d myPose = robotPose;
+    Pose2d targetPose = goalPose;
+    double targetDistance = 90;

```

Comment:
Why hard code the distance?

Reply:
Just because I know this distance was a good distance with a soft shot. Please free to change this to a better way if you have one. I think we have to look at this code and update it quite a bit, but with only 2 days left I don't know how we make all of these changes. These are all just stop gaps for Lansing and after that we can clean the code up. Let me know if you agree

Reply:
My only fear is that I think several auton paths will be broken.  I'll see if I can get some time this PM and I'll create a branch off of your branch.   Then make the proposed changes, if you don't mind, then we can test again with that branch to make sure things didn't get broken.  Let me know what you think.

Reply:
Yeah, that is a great idea! We can even run my branch with autons and see what happens